src/main/kotlin/racing/view/OutputView.kt

```diff
@@ -1,24 +1,33 @@
 package racing.view
 
 import racing.domain.Car
-
-private const val POSITION_CHAR = "-"
+import racing.domain.RacingCars
 
 internal class OutputView {
 
-    fun renderStartMessage() {
+    fun renderResult(racingCarsResult: List<RacingCars>, winners: List<Car>) {
         println("\n실행 결과")
+
+        racingCarsResult.forEach() {
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        racingCarsResult.forEach {
```